tests-e2e/cypress/integration/channels/rhs/status_update_spec.js

```diff
@@ -241,7 +242,39 @@ describe('channels > rhs > status update', () => {
                 cy.get('#confirm-modal-light').should('not.exist');
             });
 
-            it('discard explicitily', () => {
+            it.only('click overview link, go back and save', () => {
+                // # Run the `/playbook update` slash command.
+                cy.executeSlashCommand('/playbook update');
+
+                // # Get dialog modal.
+                cy.getStatusUpdateDialog().within(() => {
+                    // # Type the invalid data
+                    cy.findByTestId('update_run_status_textbox').clear().type('My valid and important changes that I don\'t want to lose');
+
+                    // * Click overview link
+                    cy.findByTestId('run-overview-link').click();
+                });
+
+                // * Go back from unsaved changes modal
+                cy.get('#confirm-modal-light').within(() => {
+                    cy.findByTestId('modal-cancel-button').click();
+                });
```

Comment:
This does test that the modal is open, but what about some specific check, as in:

```suggestion
                // * Verify that the confirmation modal is shown
                cy.get('#confirm-modal-light').should('exist').within(() => {
                    // # Go back from unsaved changes modal
                    cy.findByTestId('modal-cancel-button').click();
                });
```